Description: drop feature cleanup-markdown
 The cleanup-markdown feature depends on pulldown-cmark-to-cmark, which is not
 packaged in Debian. Drop the feature to make the dependencies of
 librust-prost-build-dev satisfiable.
Author: Emanuele Rocca <ema@debian.org>
Last-Update: 2023-01-26
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: prost-build/Cargo.toml
===================================================================
--- prost-build.orig/Cargo.toml
+++ prost-build/Cargo.toml
@@ -59,15 +59,6 @@ default-features = false
 version = "0.11.9"
 default-features = false
 
-[dependencies.pulldown-cmark]
-version = "0.9.1"
-optional = true
-default-features = false
-
-[dependencies.pulldown-cmark-to-cmark]
-version = "10.0.1"
-optional = true
-
 [dependencies.regex]
 version = "1.5.5"
 features = [
@@ -92,8 +83,4 @@ version = "0.7"
 default-features = false
 
 [features]
-cleanup-markdown = [
-    "pulldown-cmark",
-    "pulldown-cmark-to-cmark",
-]
 default = ["syn"]
